README.md

```diff
@@ -1,12 +1,4 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Ryan Trinh's portfolio and SPS projects
```

Comment:
Nitpick: Add a "." so that it's clear that the commit wasn't cutoff and the complete README was committed.

Reply:
Add a "." in new commit.